tablo/models.py

```diff
@@ -542,30 +581,30 @@ def get_quantile_breaks(self, field, break_count):
 
     def get_natural_breaks(self, field, break_count):
 
-        if not self._validate_fields([field]):
-            raise ValueError('Invalid field')
-
-        sql_statement = """SELECT service_table.{field} FROM
-                            (SELECT ROW_NUMBER() OVER (ORDER BY {field}) AS row_number,
```

Comment:
Those changes look good to me. I like how the tables are broken out more clearly.